src/components/fields/derived.js

```diff
@@ -78,6 +78,19 @@ export const AxisOverlayDropdown = connectToContainer(UnconnectedDropdown, {
   },
 });
 
+export const RangesliderVisible = connectToContainer(UnconnectedRadio, {
+  modifyPlotProps: (props, context, plotProps) => {
+    if (
+      !plotProps.isVisible &&
+      context.fullContainer._id &&
+      context.fullContainer._id.startsWith('x')
```

Comment:
note to self: exclude 3d traces somehow, also should provide some message for multi and y axis selection

Reply:
Vera says to use `_axisGroup` ... note to self 2: should exclude Y and All from AxesSelector where appropriate, and hide the whole panel where appropriate

Reply:
will be done as part of #424